server/database/config/fakeData.sql

```diff
@@ -53,3 +57,6 @@ INSERT INTO exercises (exercise_name, met) VALUES
 ('aerobic', 8.0),
 ('slide board',11.0),
 ('slimnastics', 6.0);
+
+
+INSERT INTO UserFoodRelation(users_id,food_id,food_category_id,amount_in_gram) values (1,1,1,3);
```

Comment:
Please review the problem of new line in each of your files

Reply:
Ok :+1: